### PR TITLE
feat(ui-mode): add annotations tab

### PR DESCRIPTION
**Pull request by yury-s**
image

### CODE REVIEW

File: packages/trace-viewer/src/ui/workbench.tsx [hunk 1]
```diff
@@ -223,6 +225,17 @@ export const Workbench: React.FunctionComponent<{
     sourceTab,
     attachmentsTab,
   ];
+
+  if (annotations !== undefined) {
+    const annotationsTab: TabbedPaneTabModel = {
+      id: 'annotations',
+      title: 'Annotations',
+      count: annotations?.length ?? 0,
```
Comment (dgozman) [inline packages/trace-viewer/src/ui/workbench.tsx:233]:
```suggestion
      count: annotations.length,
```

File: packages/trace-viewer/src/ui/annotationsTab.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+import * as React from 'react';
+import './annotationsTab.css';
+import { PlaceholderPanel } from './placeholderPanel';
+import { linkifyText } from '@web/renderUtils';
+
+type Annotation = { type: string; description?: string; };
+
+export const AnnotationsTab: React.FunctionComponent<{
+  annotations: Annotation[] | undefined,
```
Comment (dgozman) [inline packages/trace-viewer/src/ui/annotationsTab.tsx:25]:
```suggestion
  annotations: Annotation[],
```

File: packages/trace-viewer/src/ui/annotationsTab.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+}> = ({ annotations }) => {
+
+  if (!annotations || !annotations.length)
```
Comment (dgozman) [inline packages/trace-viewer/src/ui/annotationsTab.tsx:28]:
```suggestion
  if (!annotations.length)
```

**Review by dgozman — APPROVED** [review]

**Comment by cassus** [comment]
This is awesome! Do you also plan to add this to the trace viewer?

**Pull request merged**